.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,84 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Unit & Integration Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-version: [ '8.0' ]
+        coverage-driver: [ pcov, xdebug ]
+        include:
+          - { operating-system: 'windows-latest', php-version: '8.0', coverage-driver: 'xdebug' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.0', coverage-driver: 'pcov' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.0', coverage-driver: 'xdebug' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.1', coverage-driver: 'pcov' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.1', coverage-driver: 'pcov' }
+          - { operating-system: 'ubuntu-latest', php-version: '8.2', coverage-driver: 'xdebug' }
+
+    name: CI on ${{ matrix.operating-system }} with PHP ${{ matrix.php-version }}, using ${{ matrix.coverage-driver }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Configure for PHP >= 8.2
+        if: "matrix.php-version >= '8.2'"
+        run: |
+          composer config platform.php 8.1.99
```

Comment:
is it still needed? cc @sidz

Reply:
I think we could rework this to use `--ignore-platform-req=php+` instead, but I would prefer to do this later. I've tried to re-use as much as possible `ci.yaml` and rework the different steps later

Reply:
@maks-rafalko we don't need it anymore as php 8.2 has been released.

also this section has been removed in https://github.com/infection/infection/pull/1775/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddL37-L41